From f7205db09b6471e733e70ffb70f8dc4b773f46b7 Mon Sep 17 00:00:00 2001
From: jkreierh <jkreierh@90b65887-3827-0410-9a23-83215b262276>
Date: Tue, 22 Jan 2019 15:34:42 +0000
Subject: [PATCH 0941/1076] MANTIS 53235 Mantis 53235: New Baggage-Handling
 status BAGGAGE_INTERFERENCE_STAFF

ConveyorDev interface 1.17

Support the "Blue Button" in the Denver SBD setup. A transport to the verify position will be inhibited as long as the blue button is pressed. The BAGGAGE_INTERFERENCE_STAFF code is signalled to the application in order to show an explanation about the slight delay.


git-svn-id: svn://localhost/SelfServiceCommon/trunk@1425 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl b/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
index b973d167..74996ffa 100644
--- a/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
@@ -6,7 +6,7 @@
  * Contains module conveyordev, which is the platform-internal interface
  * definition for the massai conveyor device component.
  *
- * @author Jï¿½rgen Kreierhoff
+ * @author Juergen Kreierhoff
  *
  **************************************************************************
 **/
@@ -39,7 +39,7 @@
 **/
 module conveyordev
 {
-   const string interface_version = "1.16";
+   const string interface_version = "1.17";
 /*
 using DeviceComponents::indicationListener and DeviceComponents::Generic from mdci.idl
   RC indicationListener::indication(in any ind);
@@ -194,7 +194,8 @@ using DeviceComponents::indicationListener and DeviceComponents::Generic from md
       ERR_UPRIGHT_BAG,              /**< Detected bag stands upright */
       ERR_SLOPED_BAG,               /**< Bag leaning/levering on side walls probnably reducing weight */
       ERR_VOLUME_NOT_DETERMINABLE,  /**< The SBD system cannot perform a volumetric analysis of the bag */
-      ERR_OVERFLOW_TUB              /**< Detected bag overflows the sides of the tub */
+      ERR_OVERFLOW_TUB,             /**< Detected bag overflows the sides of the tub */
+      ERR_STAFF_INTERFERENCE        /**< Temporary staff interference suspends baggage handling */
    };
 
 
-- 
2.41.0.windows.1

